syntax = "proto2";
package util.proto;

message ObjectId {
  optional uint64 type = 1;
  optional uint64 number = 2;
  optional string tag = 3;
}
